<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrometerEEPROM</class>
 <widget class="QDialog" name="SpectrometerEEPROM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>485</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EEPROM</string>
  </property>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>450</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_read">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>450</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_write">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_backupCreate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_backupRestore">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>450</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Structure length</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>5</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Structure version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_structureLength">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_structureVersion">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>5</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_information">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>35</y>
     <width>520</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>5</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>bytes</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>620</width>
     <height>370</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Detector</string>
    </attribute>
    <widget class="QLineEdit" name="detectorType">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Detector type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of pixels</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_pNumPixels">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>65</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>135</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Intercept</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_wavelengthCalibration_intercept">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>135</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_wavelengthCalibration_X1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_wavelengthCalibration_X2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>185</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>235</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_wavelengthCalibration_X3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_wavelengthCalibration_X4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>235</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wavelength calibration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Non-linearity calibration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>140</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_intercept">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>115</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X1">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>65</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>65</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Intercept</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>115</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>165</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>165</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X6</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X6">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>215</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X7</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_X7">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>215</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_lowCounts">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>250</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Low counts</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonlinearityCalibration_highCounts">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>275</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>275</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>High counts</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_nonlinearityCorrection_enabled">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable non-linearity correction</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Defective pixels</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_defectivePixels">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>141</width>
       <height>96</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>65</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Offset 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>125</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Offset 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ext. Offset</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_extOffset">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_offset2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>125</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_offset1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_gain2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>65</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_gain1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gain and offset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Ethernet</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_IpAddr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_NetMask">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>35</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gateway</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Net mask</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_TcpPort">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>85</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_Gateway">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="m_DhcpEnabled">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DHCP enabled</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP address</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Software</string>
    </attribute>
    <widget class="QLineEdit" name="a_pFPGAVersion">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FPGA version</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Firmware version</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_pDLLVersion">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DLL version</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_pFirmwareVersion">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>35</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>20</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
